rush-plugins/rush-azure-storage-build-cache-plugin/src/AzureAuthenticationBase.ts

```diff
@@ -80,30 +81,18 @@ export type AzureEnvironmentName = keyof typeof AzureAuthorityHosts;
 /**
  * @public
  */
-export type LoginFlowType = 'DeviceCode' | 'InteractiveBrowser' | 'AdoCodespacesAuth';
+export type LoginFlowType = 'ChainedCredential' | 'DeviceCode' | 'InteractiveBrowser' | 'AdoCodespacesAuth';
 
 /**
  * @public
  */
 export interface IAzureAuthenticationBaseOptions {
   azureEnvironment?: AzureEnvironmentName;
   credentialUpdateCommandForLogging?: string | undefined;
-  loginFlow?: LoginFlowType;
   /**
-   * A map to define the failover order for login flows. When a login flow fails to get a credential,
-   * the next login flow in the map will be attempted. If the login flow fails and there is no next
-   * login flow, the error will be thrown.
-   *
-   * @defaultValue
-   * ```json
-   * {
-   *   "AdoCodespacesAuth": "InteractiveBrowser",
-   *   "InteractiveBrowser": "DeviceCode",
-   *   "DeviceCode": null
-   * }
-   * ```
+   * @defaultValue 'ChainedCredential'
    */
-  loginFlowFailover?: Record<LoginFlowType, LoginFlowType | undefined>;
```

Comment:
This is a breaking change, can we instead construct an alternate `ChainedCredential` that only contains the listed types if this parameter is used?